### PR TITLE
Assignment07 objects completed

### PR DESCRIPTION
**Pull request by saurabh-nagre**
Review needed, In assignment 07 folder

### CODE REVIEW

File: assignment07/script.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        }
+
+        console.log(obj1)
+        console.log(obj2)
+        
+        if(obj1==obj2){
+            console.log("obj1 is same as obj2")
+        }
+        else {
+            // still giving this, why?
```
Comment (Surajrane45) [inline assignment07/script.js:40]: You can't compare objects using ==  operator as objects/arrays are passed & handled in JS using their reference address. You can use JSON.stringify(obj1) === JSON.stringify(obj2) as the easiest method or else use deepCompare function of lodash library

File: assignment07/script.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+keys.forEach((key)=>{
+    output[key] = obj[key]
+})
+
+console.log(output)
+// { a: 'Apple', d: 'Dog' }
+function filterObject(obj){
+    
+    return Object.keys(obj).filter((key)=>{
```
Comment (Surajrane45) [inline assignment07/script.js]: You could have created the new object here itself & avoided an extra loop

File: assignment07/script.js (hunk 1 above)
Comment (saurabh-nagre) [reply assignment07/script.js:40]: Understood

File: assignment07/script.js (hunk 2 above)
Comment (saurabh-nagre) [reply assignment07/script.js]: Implemented using reduce, thanks

**Pull request closed without merging**